### PR TITLE
#166841003 Integrate coveralls code coverage service 

### PR DESCRIPTION
**Pull request by Alpha1202**
#### What does this PR do?

Integrate coveralls code coverage service
Install nyc for code reporting

#### Description of Task to be completed?

integrate ah-nyati-backend on coveralls.io
Install coveralls
install nyc
add coverall badge for test coverage
create .coverall.yml file
write coverall script on package.json
write coveralls-coverage script on package.json

#### How should this be manually tested?

After running the test on your local machine,
Check the coverall badge on the readme.md for the test coverage.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#166841003
https://www.pivotaltracker.com/story/show/166841003

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A

### CODE REVIEW

**Review by Cavdy — APPROVED** [review]
All good..

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: zdKtmEe7zc0wg8GHKoXRbwfaT5OwLe63F
```
Comment (phembarl) [inline .coveralls.yml]: Please add an extra line here

**Comment by phembarl** [comment]
LGTM

**Review by chokonaira — APPROVED** [review]
Good

File: lcov.info [hunk 2]
```diff
@@ -0,0 +1,72 @@
+TN:
```
Comment (fob413) [inline lcov.info]: This file should be gitignored

**Review by fob413 — CHANGES_REQUESTED** [review]
Please make the small changes required and get your teammates to review

**Review by czarjulius — APPROVED** [review]

**Review by fob413 — APPROVED** [review]

**Pull request merged**